movie.sh

```diff
@@ -1,12 +1,18 @@
 #!bin/bash
+
 function getrating()
 {
-curl "http://www.omdbapi.com/?t=$1&y=&plot=short&r=xml" > mov.txt
+#Fetch tuple from the Omdb API
```

Comment:
All comments should be on the same level of indentation as the code it refers to. All comments must start with # (i.e. a hash symbol and a space).